### PR TITLE
Feat: support LineString and MultiLineString

### PR DESCRIPTION
**Pull request by imkonsowa**
## Summary
 Add support for ClickHouse `LineString` and `MultiLineString` geo column types.

  - `LineString` maps to `orb.LineString` (stored as `Array(Point)` per [ClickHouse docs](https://clickhouse.com/docs/sql-reference/data-types/geo#linestring))
  - `MultiLineString` maps to `orb.MultiLineString` (stored as `Array(LineString)`)

  This completes geo type coverage alongside existing Point, Ring, Polygon, and MultiPolygon types.

## Checklist
Delete items not relevant to your PR:
- [x] Unit and integration tests covering the common scenarios were added

### CODE REVIEW

**Comment by imkonsowa** [comment]
Hello @kavirajk @SpencerTorres Would you please have a look at this PR?

File: examples/clickhouse_api/geo.go [hunk 1]
```diff
@@ -21,9 +22,11 @@ func GeoInsertRead() error {
 		CREATE TABLE example (
 				point Point,
 				ring Ring,
+				lineString LineString,
```
Comment (kavirajk) [inline examples/clickhouse_api/geo.go:25]: Thanks for updating the example. Can you also add this example in `examples/std/` interface as well?

File: tests/geo_linestring_test.go [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+		) Engine MergeTree() ORDER BY tuple()
+		`
+		defer func() {
+			conn.Exec(ctx, "DROP TABLE IF EXISTS test_geo_linestring")
+		}()
+		require.NoError(t, conn.Exec(ctx, ddl))
+		batch, err := conn.PrepareBatch(ctx, "INSERT INTO test_geo_linestring")
+		require.NoError(t, err)
+		var (
+			col1Data = orb.LineString{
```
Comment (kavirajk) [inline tests/geo_linestring_test.go:42]: Can you add two more edge cases?
1. Empty Point
2. Single Point

Also on other tests.

**Review by kavirajk — COMMENTED** [review]
Thanks @imkonsowa for the PR. Implementation looks good. Added two minor comments. Happy to merge after that.

File: tests/geo_linestring_test.go (hunk 2 above)
Comment (imkonsowa) [reply tests/geo_linestring_test.go:42]: Thanks @kavirajk Your comments are addressed and pushed to the MR.

**Review by kavirajk — APPROVED** [review]

**Comment by imkonsowa** [comment]
Thanks @kavirajk for reviewing
I will use the head now, but would like to use a tag instead do you have any estimation of when next tag will be created? Thanks!

**Comment by kavirajk** [comment]
@imkonsowa I'm planning to do one release by end of Jan.
